### DISCUSSION TITLE
Working on an updated Wiki for Kolbot

### DISCUSSION THREAD

**Discussion by InterestingIndeed** [blizzhackers/kolbot#331] [Ideas]
I would love to update the Wiki here. From my experience in the old repository, a lot of questions repeat themselves.  Some explanations are written poorly or are missing out on some stuff. An updated Wiki could help with that.

**Comment by jaenster** [comment]
@mf022

**Comment by mf022** [comment]
wiki is written directly on [documentation](https://github.com/blizzhackers/documentation) and [pickits](https://github.com/blizzhackers/pickits) repositories.

The formatting is the same Markdown, but it's easier to update the pages. You could make all the changes offline and to add them in 1 commit.

@InterestingIndeed 
I'll sent you an invite to the base team. Check your email.

**Comment by mf022** [comment]
unfortunately, there were no improvements made by @InterestingIndeed, so he was removed from bh team
